modules/custom/mentions/src/Plugin/Mentions/Entity.php

```diff
@@ -40,17 +43,43 @@ public static function create(ContainerInterface $container, array $configuratio
     $token = $container->get('token');
     $entity_manager = $container->get('entity.manager');
     $entity_query = $container->get('entity.query');
+    $config = $container->get('config.factory');
     return new static(
       $token,
       $entity_manager,
-      $entity_query
+      $entity_query,
+      $config
     );
   }
 
   /**
    * {@inheritdoc}
    */
   public function outputCallback($mention, $settings) {
+    // If the mentions is run with cron, replace the output ourself.
+    if (PHP_SAPI === 'cli') {
+      // Get the profile.
+      $profile = $this->entityManager->getStorage($mention['target']['entity_type'])
+        ->load($mention['target']['entity_id']);
+
+      // Get the output value according to the config settings.
+      $config = $this->config->get('mentions.settings');
+      switch ($config->get('suggestions_format')) {
+        case SOCIAL_PROFILE_SUGGESTIONS_FULL_NAME:
+        case SOCIAL_PROFILE_SUGGESTIONS_ALL:
+          $output['value'] = $profile->getOwner()->getDisplayName();
```

Comment:
Doesn't this already fall back to the account name? 

``` $name = $this
    ->getAccountName() ?: \Drupal::config('user.settings')
    ->get('anonymous');
  \Drupal::moduleHandler()
    ->alter('user_format_name', $name, $this);
  return $name;``` 

Or did we alter it empty somewehere? 🤔